include/picoprobe_config.h

```diff
@@ -29,33 +29,39 @@
 
 #define INCLUDE_RTT_CONSOLE
 
+#if defined(OPT_SERIAL_CRLF)
+    #define CRLF "\r"
```

Comment:
could you please use a better name which is not prone to collision with, something like _OPT_CR

Reply:
Fixed